### PR TITLE
helm chart documentation overhaul

### PR DESCRIPTION
**Pull request by tuffacton**
Documented just about everything in the Harness helm charts. Please re-run helm-docs to populate the values in the README

### CODE REVIEW

**Comment by arya-harness** [comment]
We should also added documentations for override.yaml as well.

**Comment by pratmit** [comment]
Yes, this is for values.yaml. We should definitely create a separate one for override.yaml.

**Review by arya-harness — APPROVED** [review]
Looks good.

File: src/harness/values.yaml [hunk 1]
```diff
@@ -109,70 +120,85 @@ global:
...
+# -- Install the Continuous Integration (CI) manager pod
 ci:
   ci-manager:
     affinity: {}
     nodeSelector: {}
     tolerations: []
 
+# -- Config for Security Test Orchestration (STO) 
 sto:
+  # -- Install the STO core
```
Comment (arya-harness) [inline src/harness/values.yaml:132]: These are not installing the STO core.
We can call them: "Overrides for STO-core"

**Comment by tuffacton** [comment]
When we get an opportunity can we run a `helm-docs` on this new values chart so we can update the README more clearly?

**Review by pratmit — COMMENTED** [review]
LGTM

**Review by pratmit — APPROVED** [review]

**Pull request merged**